roofit/roofitcore/src/RooFitDriver.cxx

```diff
@@ -359,9 +370,11 @@ void RooFitDriver::computeCPUNode(const RooAbsArg *node, NodeInfo &info)
 /// Returns the value of the top node in the computation graph
 double RooFitDriver::getVal()
 {
+   ++_getValInvocations;
+
```

Comment:
I think you mean here first 3 nll evaluations

Reply:
Right, but it doesn't matter now because I could revert that change anyway.